instance/models/instance.py

```diff
@@ -94,6 +96,8 @@ class Instance(ValidateModelMixin, TimeStampedModel):
     STOPPED = 'stopped'
     TERMINATING = 'terminating'
 
+    ERR_PROVISIONING_FAILED = 'provisioning_failed'
```

Comment:
Definitely I think the way you are describing it is better. FTR I thought that my approach would be good because we could retain the error after a provisioning even if we reboot the server; but rethinking this I agree that after a reboot, and having the server at a BOOTED status, it only matters if there was an error during the booting and not previous provisioning failures.

I'm going ahead and make the changes. Thanks!